AzureMonitorAgent/ama_tst/modules/install/check_ama.py

```diff
@@ -1,22 +1,20 @@
 import re
 import requests
-from bs4 import BeautifulSoup
+from lxml import html
```

Comment:
Unfortunately `lxml` runs into the same issue we had with `bs4` where it needs a separate install. Could you try using [the `xml.dom.minidom` library](https://docs.python.org/3/library/xml.dom.minidom.html) instead? I've confirmed it exists without extra install on base Python for 2.7 and 3.6.9, but beyond that could you verify it does everything it needs to do?